libnetwork/drivers/bridge/setup_bridgenetfiltering.go

```diff
@@ -45,13 +45,24 @@ func setupIPv6BridgeNetFiltering(config *networkConfiguration, _ *bridgeInterfac
 	return nil
 }
 
+func loadBridgeNetFilterModule() error {
+	fullPath := "/proc/sys/net/bridge/bridge-nf-call-iptables"
```

Comment:
I notice that `enableBridgeNetFiltering` is called with `nfParam` parameter that.. is the same as this, but has either `iptables` or `ip6table`; 


```bash
git grep 'enableBridgeNetFiltering'
libnetwork/drivers/bridge/setup_bridgenetfiltering.go:          return enableBridgeNetFiltering("/proc/sys/net/bridge/bridge-nf-call-iptables")
libnetwork/drivers/bridge/setup_bridgenetfiltering.go:          return enableBridgeNetFiltering("/proc/sys/net/bridge/bridge-nf-call-ip6tables")
libnetwork/drivers/bridge/setup_bridgenetfiltering.go:func enableBridgeNetFiltering(nfParam string) error {
```


Should this function unconditionally use `/proc/sys/net/bridge/bridge-nf-call-iptables`, or should it take an argument and get called passing through the `nfParam` from `enableBridgeNetFiltering` ?

Reply:
Good point. When `br_netfilter` module gets loaded, both these files will be present under `/proc/sys/net/bridge/`
I will make the adjustments, thanks.